lib/app-routes/users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
This uses a new file structure paradigm that I'm trying out: the `app-routes` simply register routes which are defined in each `controller`. This will make more sense as things become more complex.